service/http.proxy/pom.xml

```diff
@@ -77,6 +77,11 @@
             <artifactId>specification.http</artifactId>
             <scope>test</scope>
         </dependency>
+        <dependency>
```

Comment:
I'm not sure if your code requires this change, or if this is a side affect of catchup merge, or if this is not needed at all...?